pandas/conftest.py

```diff
@@ -376,19 +376,25 @@ def unique_nulls_fixture(request):
              FixedOffset(0), FixedOffset(-300), timezone.utc,
              timezone(timedelta(hours=1)),
              timezone(timedelta(hours=-1), name='foo')]
+TIMEZONE_IDS = ['None', 'UTC', 'US/Eastern', 'Asia/Tokyp',
```

Comment:
pls don’t merge things until i look 

this is duplicating too much code here

Reply:
what is the point of of these extras?

Reply:
@jreback : This is carried over from #25637, and the justification was given as a link in the PR description.  I leave it here below for convenience:

https://github.com/pandas-dev/pandas/pull/25637#discussion_r289441644

Reply:
maybe so but that other has not been reviewed at all

Reply:
this is not done in a nice way

these should simply be tuples of timezone and id and unpacked when needed

can u do a PR to fix

Reply:
and to be honest these may not even be needed to do it like this it’s a lot more verbose

Reply:
@jreback : The unpacking option seems to overcomplicate as you mentioned.  Alternatively, we can just do a list comprehension and pull the `__repr__` (or `__str__`) on the elements of `TIMEZONES`.  That would still keep the IDs but be less "duplicative".

Reply:
yes i think a nice function would be fine

Reply:
> @jreback: this is not done in a nice way

I disagree. Compare the [outputs](https://github.com/pandas-dev/pandas/pull/26600#issuecomment-497927487) of @gfyoung's PR with the state I introduced.

I made the IDs explicit precisely because the strs/reprs yields a useless, ugly mix of stuff. Saving these 4 lines is not worth the added hassle when actually debugging tz-tests with `pytest -v`.